cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java

```diff
@@ -157,20 +157,10 @@ public Boolean call() throws Exception {
         if (!getMonitorClient().allSystemServicesOk()) {
           return false;
         }
-        // Check that the dataset service is up, and also that the default namespace exists
-        // Using list and checking if default namespace exists, as opposed to using get()
-        // so we don't have to unnecessarily add a try-catch for NamespaceNotFoundException, since that exception is
-        // not handled in checkServicesWithRetry.
-        List<NamespaceMeta> namespaces = getNamespaceClient().list();
-
-        if (namespaces.size() == 0) {
-          return false;
-        }
-        if (namespaces.contains(NamespaceMeta.DEFAULT)) {
-          return true;
-        }
-        throw new IllegalStateException("Default namespace not found. Instead found unexpected namespaces: "
-                                          + namespaces);
+        // list() method will throw exception if the http request does not receive response 
```

Comment:
I think you should still have the comment
"Check that the dataset service is up", so its clear why we're calling `list()`